src/app/core/models/customer/customer.interface.ts

```diff
@@ -17,6 +17,7 @@ export interface CustomerData extends Customer {
   phoneMobile?: string;
   fax?: string;
   email?: string;
+  login?: string;
```

Comment:
why do you introduce the login field here, it is always the same as email and can be removed
a customer has no login - only the user has it!!!